### DISCUSSION TITLE
projection or forecast?

### DISCUSSION THREAD

**Discussion by avallecam** [epiforecasts/bpmodels#63] [Q&A] [answered]
Great [vignette for projecting incidence](https://epiverse-trace.github.io/bpmodels/articles/projecting_incidence.html)!

The vignette is consistent with the usage of the word "projection" in most of the document, except at the end of the second paragraph where it says "forecast an outbreak". In the [referenced paper](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.18.2000543), after a quick skim, my initial impression is that the authors use both of them interchangeably, e.g. for the projection/forecast time intervals.

Could we use them interchangeably?

My last reference about this is the one below:

> What do I mean by projections/forecasts/predictions?
> 
> - Projections: short term not mechanistic – taking current trend and continuing
> - Forecasts: relies on somehow more mechanistic model but typically assumes conditions in future remain stable
> - Predictions: relies on understanding the system and making hypothesis about future conditions – closer scenario modelling
> 
> reference: https://www.reconlearn.org/post/projections.html

I would like to know if I can be flexible with the terminology given its usage in the field, or if they do refer to different things and then need to homogenize the terminology.

**Comment by jamesmbaazam** [comment]
Great observation, Andree!

As you have noticed, the terminologies can sometimes be confusing. Some authors tend to be less strict and use them interchangeably. However, based on the RECON definition, I believe the vignette is about "forecasting" rather than "projection" since the underlying model is somewhat mechanistic and also assumes that conditions will remain the same throughout the forecasting window. Joel Hellewell actually has [a nice reflection](https://jhellewell14.github.io/2021/11/16/forecasting-projecting.html) on this distinction that you might find interesting.

I would like @sbfnk to weigh in on what is the appropriate terminology to use here.

**Comment by sbfnk** [comment, marked as answer]
I think sadly there isn't any commonly accepted definition. To me these are quite similar and I would perhaps define them as:
- Prediction == forecast (except in a statistical sense when predictions can be any unobserved, including past, data) 
- Forecast: a statement about the future i.e. giving predicted probabilities for future outcomes (if a deterministic forecast then that probability == 1)
- Projection: the same as "simulation", i.e. a model run forward; often, but not necessarily always, assuming things may stay the same. A projection may be used as a forecast, or as a scenario.

**Comment by jamesmbaazam** [comment]
Thanks for your input, Seb.

**Comment by jamesmbaazam** [comment]
@Andree, I think this might be better placed under the Discussions, so I will move it there.
